tools/dep_updaters/update-postject.sh

```diff
@@ -20,3 +20,9 @@ NPM="$ROOT/deps/npm/bin/npm-cli.js"
 "$NODE" "$NPM" init --yes
 
 "$NODE" "$NPM" install --no-bin-links --ignore-scripts postject
+
+cd ../../..
+rm -rf deps/postject
+mkdir deps/postject
+cp test/fixtures/postject-copy/node_modules/postject/LICENSE deps/postject
+cp test/fixtures/postject-copy/node_modules/postject/dist/postject-api.h deps/postject
```

Comment:
I think we should add a $WORKSPACE just like the other updater scripts.

Reply:
That's an improvement that could be done in a separate pr, right? I'm trying to focus this pr on addressing Michael's comment that's linked in the pr description

Reply:
It can be done in a different pull request if we have a TODO in here. I'm mostly raising my concerns due to the line `cd ../../..`. 

If you think it would be best to make these changes on a different pull request, can you add a TODO, and resolve my comment?

Reply:
That sounds good to me. Could u add a code suggestion using the github? I'd be happy to commit that in. (I'm asking u to suggest it because I'm on my mobile now, where it's hard to do :p)

Reply:
Done!

Reply:
Thanks!